inc/class-sitemaps-provider.php

```diff
@@ -46,24 +46,44 @@ class Core_Sitemaps_Provider {
 	public $slug = '';
 
 	/**
-	 * Get content for a page.
+	 * Get a URL list for a post type sitemap.
 	 *
 	 * @param string $object_type Name of the object_type.
-	 * @param int    $page_num Page of results.
-	 *
-	 * @return int[]|WP_Post[] Query result.
+	 * @param int    $page_num    Page of results.
+	 * @return array $url_list List of URLs for a sitemap.
 	 */
-	public function get_content_per_page( $object_type, $page_num = 1 ) {
-		$query = new WP_Query();
+	public function get_url_list( $object_type, $page_num = 1 ) {
```

Comment:
Based on the discussions today  this function can be refactored to remove the $object_type parameter as it's retrievalbe via $this->object_type. The $page_num parameter is no longer optional as we always pass in an integer.